pnpm-workspace.yaml

```diff
@@ -1,5 +1,6 @@
 packages:
   - "frontend/apps/*"
   - "frontend/packages/*"
+  - "frontend/packages/__mocks__/*"
```

Comment:
I was thinking that perhaps it might be easier to understand packages that are not published if they were separated as `frontend/internal-packages`.
What do you think?


```
.
└── frontend/
    ├── packages/
    │   ├── cli
    │   └── db-structure
    └── internal-packages/
        ├── jobs
        ├── db
        └── node-gtts
```

Reply:
Does that mean the `frontend/packages/*` directory only contains the following four packages?

```
[liam]$ git grep -l '"access": "public"'
frontend/packages/cli/package.json
frontend/packages/db-structure/package.json
frontend/packages/erd-core/package.json
frontend/packages/ui/package.json
```

If so, that seems like a reasonable approach to me.

Reply:
That's right! 👍🏻

Reply:
👍
No matter what, the diff is going to be enormous, so we'll likely need to open a separate PR.

Reply:
wrote the new issue  https://github.com/liam-hq/liam/issues/1077

Reply:
Thanks 👍🏻